tests/framework/icd/test_icd.cpp

```diff
@@ -576,7 +576,14 @@ VKAPI_ATTR VkResult VKAPI_CALL test_vkGetPhysicalDeviceSurfacePresentModesKHR(Vk
 // 1.0
 VKAPI_ATTR void VKAPI_CALL test_vkGetPhysicalDeviceFeatures(VkPhysicalDevice physicalDevice, VkPhysicalDeviceFeatures* pFeatures) {}
 VKAPI_ATTR void VKAPI_CALL test_vkGetPhysicalDeviceProperties(VkPhysicalDevice physicalDevice,
-                                                              VkPhysicalDeviceProperties* pProperties) {}
+                                                              VkPhysicalDeviceProperties* pProperties) {
+    if (pProperties) {
+        auto& phys_dev = icd.GetPhysDevice(physicalDevice);
+        strncpy(pProperties->deviceName, phys_dev.deviceName.c_str(), VK_MAX_PHYSICAL_DEVICE_NAME_SIZE);
```

Comment:
I think windows will complain about using strncpy. Should be possible to use plain `std::copy` here, there are other places where I copy strings in/out of C char arrays.

Reply:
We actually use strncpy all over the loader.  So we must have that warning disabled.

Reply:
Test framework, not loader code. But yes, it probably is disabled for the loader itself. Definitely a nitpick, not a big deal.

Reply:
Good point, I'll change it.